src/Form/Type/SecurityRolesType.php

```diff
@@ -135,7 +137,7 @@ public function configureOptions(OptionsResolver $resolver): void
         ]);
 
         // Symfony 2.8 BC
-        if ($resolver->isDefined('choices_as_values')) {
+        if (method_exists(FormTypeInterface::class, 'setDefaultOptions')) {
             $resolver->setDefault('choices_as_values', true);
```

Comment:
The coverage decreased and coveralls says this is no longer covered by tests… is that true and normal?

Reply:
It is covered on 2.8 https://coveralls.io/jobs/33261397/source_files/324132516#L140, because `setDefaultOptions` exist only there.